Formula/l/lume.rb

```diff
@@ -0,0 +1,21 @@
+class Lume < Formula
+  desc "Create and manage Apple Silicon-native virtual machines"
+  homepage "https://github.com/trycua/lume"
+  url "https://github.com/trycua/lume/archive/refs/tags/v0.1.6.tar.gz"
+  sha256 "4f3216eef9c2a805cbbd3893efef3e161e00ad76718a00623c0481d2c2855767"
+  license "MIT"
+  head "https://github.com/trycua/lume.git", branch: "main"
+
+  uses_from_macos "swift" => :build
+  depends_on :macos
+
+  def install
+    system "swift", "build", "--disable-sandbox", "-c", "release", "--product", "lume"
```

Comment:
We might also want to include a `codesign` invocation that will include the `com.apple.security.virtualization` entitlement.. the Tart formula was [handling that like this](https://github.com/Homebrew/homebrew-core/blob/master/Formula/t/tart.rb#L38).